ocl_icd_loader.h

```diff
@@ -43,11 +43,34 @@ cl_platform_id selectPlatformID(cl_platform_id pid) {
   return getDefaultPlatformID();
 }
 
+typedef cl_uint cl_layer_info;
+typedef cl_uint cl_layer_api_version;
+#define CL_LAYER_API_VERSION 0x4240
+#define CL_LAYER_NAME        0x4241
+#define CL_LAYER_API_VERSION_100 100
+
+typedef __typeof__(clGetPlatformInfo) *clGetPlatformInfo_fn;
+CL_API_ENTRY typedef cl_int (CL_API_CALL *clGetLayerInfo_fn)(
+    cl_layer_info  param_name,
+    size_t         param_value_size,
+    void          *param_value,
+    size_t        *param_value_size_ret);
+
+CL_API_ENTRY typedef cl_int (CL_API_CALL *clInitLayer_fn)(
+    cl_uint                         num_entries,
+    const struct _cl_icd_dispatch  *target_dispatch,
+    cl_uint                        *num_entries_out,
+    const struct _cl_icd_dispatch **layer_dispatch);
+
```

Comment:
```suggestion
typedef cl_int (CL_API_CALL *clGetLayerInfo_fn)(
    cl_layer_info  param_name,
    size_t         param_value_size,
    void          *param_value,
    size_t        *param_value_size_ret);

typedef cl_int (CL_API_CALL *clInitLayer_fn)(
    cl_uint                         num_entries,
    const struct _cl_icd_dispatch  *target_dispatch,
    cl_uint                        *num_entries_out,
    const struct _cl_icd_dispatch **layer_dispatch);

```

Remove useless attribute.